<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainApp</class>
 <widget class="QMainWindow" name="MainApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>320</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>320</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>320</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MazeStudio</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="generatorGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>291</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Generator</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>291</width>
       <height>151</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="generatorFormLayout">
      <property name="horizontalSpacing">
       <number>8</number>
      </property>
      <property name="verticalSpacing">
       <number>8</number>
      </property>
      <property name="leftMargin">
       <number>16</number>
      </property>
      <property name="topMargin">
       <number>8</number>
      </property>
      <property name="rightMargin">
       <number>16</number>
      </property>
      <property name="bottomMargin">
       <number>8</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="mazeWidthLabel">
        <property name="text">
         <string>Maze Width</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="widthSpinBox">
        <property name="minimum">
         <number>2</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="singleStep">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="mazeHeightLabel">
        <property name="text">
         <string>Maze Height</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="heightSpinBox">
        <property name="minimum">
         <number>2</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="singleStep">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="algorithmsLabel">
        <property name="text">
         <string>Algorithms</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="algorithmsComboBox">
        <item>
         <property name="text">
          <string>Hunt And Kill</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QPushButton" name="generateButton">
        <property name="text">
         <string>Generate</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QPushButton" name="saveButton">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="viewerGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Viewer</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <widget class="QPushButton" name="viewerLoadButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>271</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Load (Vertex)</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="editorGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Editor</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <widget class="QPushButton" name="editorLoadButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>271</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
   </widget>
   <zorder>generatorGroupBox</zorder>
   <zorder>generatorGroupBox_2</zorder>
   <zorder>viewerGroupBox</zorder>
   <zorder>editorGroupBox</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
